packages/packages/commonwell-cert-runner/src/payloads.ts

```diff
@@ -108,3 +110,59 @@ export const mergePatient = {
   ],
   details: secondaryDetails,
 };
+
+// ORGANIZATION
+const appendOrgId = nanoid.customAlphabet("1234567890", 18)();
+const shortName: string = uniqueNamesGenerator({
```

Comment:
Orgs cant have the same name. Doesnt seem like there is a programatic way to delete orgs

Reply:
Yeah I've noticed that in the portal as well, no option to delete